tests/integration/navigation/Suggestions.test.js

```diff
@@ -117,15 +117,20 @@ describe( "Suggestions", ( ) => {
 
   // We need to navigate from MyObs to ObsEdit to Suggestions for all of these
   // tests
-  async function navigateToSuggestionsViaObsEditForObservation( observation ) {
+  async function navigateToSuggestionsViaObsEditForObservation( observation, options ) {
     const observationGridItem = await screen.findByTestId(
       `MyObservations.obsGridItem.${observation.uuid}`
     );
     await actor.press( observationGridItem );
-    const addIdButton = observation.taxon
-      ? await screen.findByLabelText( "Edit identification" )
-      : await screen.findByText( "ID WITH AI" );
-    await actor.press( addIdButton );
+    if ( options?.toTaxonSearch ) {
+      const taxonSearchButton = await screen.findByText( "SEARCH" );
+      await actor.press( taxonSearchButton );
+    } else {
+      const addIdButton = observation.taxon
+        ? await screen.findByLabelText( "Edit identification" )
+        : await screen.findByText( "ID WITH AI" );
```

Comment:
nice to have for this PR would be a unit test in `Suggestions.test.js` that checks the `hasPhotos` and `!hasPhotos` conditions for displaying this button

Reply:
Thanks, great call.